README.md

```diff
@@ -76,7 +76,7 @@ Follow these steps to train a PyTorch model with Docker on a TPU:
     distribution (only updates your pytorch-nightly conda env), run:
     ```Shell
     (vm)$ cd /usr/share/torch-xla-nightly/pytorch/xla
-    (vm)$ ./scripts/update_nightly_torch_wheels.sh
+    (vm)$ . ./scripts/update_nightly_torch_wheels.sh
```

Comment:
Why this change (sourcing it instead of running it)?

Reply:
Without it the `conda` binary couldn't be found as its sourcing happens in `/etc/profile`.